docs/config.rst

```diff
@@ -31,6 +31,15 @@ start from this file for your config::
 
     cp ~/liquidprompt/liquidpromptrc-dist ~/.config/liquidpromptrc
 
+In the event that you synchronize your configuration file across multiple
+computers, or if you have an ``/etc/liquidpromptrc`` system-wide from which
+you'd like to make minor deviations in an individual user account, you can
+augment the primary config to add in any local modifications using lines such
+as these::
+
+    LOCAL_RCFILE=$HOME/.liquidpromptrc.local
+    [ -f "$LOCAL_RCFILE" ] && source $LOCAL_RCFILE
```

Comment:
Quoting vars inside `[[` is not needed. It is needed in `[`. It is also needed when using the variable as an argument. This (and the same thing in the other file) should either be:
```suggestion
    [ -f "$LOCAL_RCFILE" ] && source "$LOCAL_RCFILE"
```
or
```suggestion
    [[ -f $LOCAL_RCFILE ]] && source "$LOCAL_RCFILE"
```
Won't be a problem unless someone has a weird char or space in their rcfile path, but it is possible.